.github/workflows/test.yml

```diff
@@ -5,8 +5,6 @@ on:
     branches:
       - master
   pull_request:
```

Comment:
do we not also need to remove line 7 for this?

Reply:
I think it would not run on pull requests at all in this case.

Reply:
I see, makes sense!